build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,275 @@
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var BBPromise = require('bluebird');
+var argv = require('minimist')(process.argv.slice(2));
+var assert = require('assert');
+var child_process = require('child_process');
+var config = require('../config');
+var extend = require('util')._extend;
+var fs = require('fs-extra');
+var git = require('gulp-git');
+var gulp = require('gulp-help')(require('gulp'));
+var request = BBPromise.promisify(require('request'));
+var util = require('gulp-util');
+
+// Set token 
+var GITHUB_ACCESS_TOKEN = process.env.GITHUB_ACCESS_TOKEN;
+var exec = BBPromise.promisify(child_process.exec);
+var gitExec = BBPromise.promisify(git.exec);
+
+var isDryrun = argv.dryrun;
+var verbose = (argv.verbose || argv.v);
+
+const issuesopt = {
+  // Url to repository 
+  url: 'https://api.github.com/repos/ampproject/amphtml/issues',
+  headers: {
+    'User-Agent': 'amp-changelog-gulp-task',
+    'Accept': 'application/vnd.github.v3+json'
+  },
+};
+
+const optionsMilestone = {
+  // Url to repository 
+  url: 'https://api.github.com/repos/ampproject/amphtml/milestones',
+  headers: {
+    'User-Agent': 'amp-changelog-gulp-task',
+    'Accept': 'application/vnd.github.v3+json'
+  },
+};
+
+if (GITHUB_ACCESS_TOKEN) {
+  issuesopt.qs = {
+    access_token: GITHUB_ACCESS_TOKEN
+  }
+  optionsMilestone.qs = {
+    access_token: GITHUB_ACCESS_TOKEN
+  }
+}
+
+function processissues() {
+  if (!GITHUB_ACCESS_TOKEN) {
+    util.log(util.colors.red('You have not set the ' +
+        'GITHUB_ACCESS_TOKEN env var.'));
+    util.log(util.colors.green('See https://help.github.com/articles/' +
+        'creating-an-access-token-for-command-line-use/ ' +
+        'for instructions on how to create a github access token. We only ' +
+        'need `public_repo` scope.'));
+    return;
+  }
+  return getGitMetadata();
+}
+
+function getGitMetadata() {
+  return getIssues().then(function() {
+    util.log(util.colors.blue('automation applied'));
+  })
+}
+/**
+ * Function goes through all the gitHub issues,
+ * gets all the Labels we are interested in,
+ * depending if missing milestone or label, 
+ * tasks applied as per design go/ampgithubautomation
+ */
+function getIssues(){
+  var promise = Promise.resolve();
+  var options = extend({}, issuesopt);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+  return request(options).then(res => {
+    const issues = JSON.parse(res.body);
+    issues.forEach(function(issue) {
+      var labels = issue.labels;
+      var issueType;
+      var milestone = issue.milestone;
+      var milestoneTitle; 
+      var milestoneState;
+      var hasPriority = false;
+      // By default we will assign 'Pending Triage' milestone, number 20 
```

Comment:
Instead of this comment, create a const called `PENDING_TRIAGE_MILESTONE` with a value of `20` at the top of this file.

Reply:
I need to keep this the way it is because I'm changing it below and then I need to change it back for the rest of the issues. let me know if it makes sense. thanks.